binaries/build.py

```diff
@@ -62,30 +57,34 @@ def build(args):
 
     os.chdir(REPO_ROOT)
 
-    ts_wheel_path = glob.glob(os.path.join(REPO_ROOT, "dist", "*.whl"))[0]
-    ma_wheel_path = glob.glob(
-        os.path.join(REPO_ROOT, "model-archiver", "dist", "*.whl")
-    )[0]
-    wa_wheel_path = glob.glob(
-        os.path.join(REPO_ROOT, "workflow-archiver", "dist", "*.whl")
-    )[0]
+    if not args.dry_run:
+        ts_wheel_path = glob.glob(os.path.join(REPO_ROOT, "dist", "*.whl"))[0]
+        ma_wheel_path = glob.glob(
+            os.path.join(REPO_ROOT, "model-archiver", "dist", "*.whl")
+        )[0]
+        wa_wheel_path = glob.glob(
+            os.path.join(REPO_ROOT, "workflow-archiver", "dist", "*.whl")
+        )[0]
+
+    else:
```

Comment:
Why are the paths different if dry_run or not?

Reply:
So if we're doing a dry_run the path REPO_ROOT/dist/*.whl doesn't actually exist so it'll be an empty list and fail when you do [0]

I think we can create a mock path so this works